pkgs/development/python-modules/imbox/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchFromGitHub, setuptools, chardet }:
+
+buildPythonPackage rec {
+  pname = "imbox";
+  version = "0.1.0";
+
+  # pypi source doesn't contain tests
+  src = fetchFromGitHub {
+    owner = "martinrusev";
+    repo = "imbox";
+    rev = "73ef2267ee6f9af927c30be68009552fc2b0bdbe";
+    sha256 = "sha256-4qMchhpkZK3cmrtRSd18PLaz/vaIE+jBIjfJnI11SWk=";
+  };
+
+  patches = [
+    ./patches/Support-to-connect-163-mail-server.patch
+    ./patches/Support-to-parse-chinese-attachment-file-name.patch
+  ];
+
+  propagatedBuildInputs = [ setuptools chardet ];
```

Comment:
`setuptools` is usually not a run-time requirement. Unless the module uses `pkg_resources` or alike.